README.md

```diff
@@ -60,6 +60,11 @@ Or to launch ghcid for `lib/command` project:
 nix-shell -A obeliskEnvs.obelisk-command --run "cd lib/command && ghcid -c 'cabal new-repl'"
 ```
 
+To re-install `ob` from source do
```

Comment:
This should also note that ob will use the version found in your ```.ob``` directory. So updating ob here, may still mean your project's ob is old.

Reply:
`.ob` ? Do you mean `.obelisk/impl` ?

Reply:
Oops, yeah that's what I meant.